desktop_version/CMakeLists.txt

```diff
@@ -295,43 +304,56 @@ target_compile_definitions(lodepng-static PRIVATE
     -DLODEPNG_NO_COMPILE_ENCODER
 )
 
+add_library(faudio-static STATIC ${FAUDIO_SRC})
+target_include_directories(
+    faudio-static PRIVATE
+    ../third_party/FAudio/include
+)
+
 if(BUNDLE_DEPENDENCIES)
     add_library(tinyxml2-static STATIC ${XML2_SRC})
     add_library(physfs-static STATIC ${PFS_SRC})
     target_compile_definitions(physfs-static PRIVATE
         -DPHYSFS_SUPPORTS_DEFAULT=0 -DPHYSFS_SUPPORTS_ZIP=1
     )
 
-    target_link_libraries(VVVVVV physfs-static tinyxml2-static lodepng-static)
+    target_link_libraries(VVVVVV physfs-static tinyxml2-static lodepng-static faudio-static)
 else()
     find_package(utf8cpp CONFIG)
 
-    target_link_libraries(VVVVVV physfs tinyxml2 utf8cpp lodepng-static)
+    target_link_libraries(VVVVVV physfs tinyxml2 utf8cpp lodepng-static FAudio)
 endif()
 
 # SDL2 Dependency (Detection pulled from FAudio)
 if(DEFINED SDL2_INCLUDE_DIRS AND DEFINED SDL2_LIBRARIES)
     message(STATUS "Using pre-defined SDL2 variables SDL2_INCLUDE_DIRS and SDL2_LIBRARIES")
     target_include_directories(VVVVVV SYSTEM PRIVATE "$<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>")
     target_link_libraries(VVVVVV ${SDL2_LIBRARIES})
+    target_include_directories(faudio-static SYSTEM PRIVATE "$<BUILD_INTERFACE:${SDL2_INCLUDE_DIRS}>")
+    target_link_libraries(faudio-static ${SDL2_LIBRARIES})
 elseif (EMSCRIPTEN)
     message(STATUS "Using Emscripten SDL2")
-    target_compile_options(VVVVVV PUBLIC -sUSE_SDL=2 -sUSE_SDL_MIXER=2)
-    target_link_libraries(VVVVVV -sUSE_SDL=2 -sUSE_SDL_MIXER=2)
+    target_compile_options(VVVVVV PUBLIC -sUSE_SDL=2)
+    target_link_libraries(VVVVVV -sUSE_SDL=2)
+    target_compile_options(faudio-static PUBLIC -sUSE_SDL=2)
```

Comment:
Anything referring to `faudio-static` should also be behind `if(BUNDLE_DEPENDENCIES)`.